src/jit/morph.cpp

```diff
@@ -2951,7 +2955,10 @@ GenTreeCall* Compiler::fgMorphArgs(GenTreeCall* callNode)
 #ifdef _TARGET_ARM_
 
         bool passUsingIntRegs;
-
+#ifdef ARM_SOFTFP
+        passUsingFloatRegs = false;
+        passUsingIntRegs   =  (intArgRegNum < MAX_REG_ARG);
+#else
         if (lateArgsComputed)
```

Comment:
OK, but this is not my first change in the file, do we have intArgRegNum correctly computed for varargs as well?